src/settings.ts

```diff
@@ -929,15 +956,15 @@ export class SRSettingTab extends PluginSettingTab {
             }),
         );
         /* 
-        // Documentation link & GitHub links
-        containerEl.createEl("hr").insertAdjacentHTML("beforeend");
-
-        // Copyright notice
-        const copyright_paragraph = containerEl.createEl("p");
-        copyright_paragraph.addClass("sr-small-font");
-        copyright_paragraph.insertAdjacentHTML("beforeend", `
-            <em>Shell commands</em> plugin Copyright &copy; 2021 - 2023 Jarkko Linnanvirta. This program comes with ABSOLUTELY NO WARRANTY. This is free software, and you are welcome to redistribute it under certain conditions. See more information in the license: <a href="${GitHub.license}">GNU GPL-3.0</a>.
-        `);     */
+		// Documentation link & GitHub links
+		containerEl.createEl("hr").insertAdjacentHTML("beforeend");
```

Comment:
Curious why the indentation has changed

Reply:
I just have run the format not sure could be undone if you like?